plugins/modules/dcnm_vrf.py

```diff
@@ -448,15 +448,21 @@
       - ip_address: 192.168.1.224
       - ip_address: 192.168.1.225
         vrf_lite:
-         # All parameters under vrf_lite except peer_vrf are optional and
-         # will be supplied by DCNM when omitted in the playbook
-          - peer_vrf: test_vrf_1 # peer_vrf is mandatory
+         # All parameters under vrf_lite are optional
```

Comment:
Should you expand this comment to indicate that while they are all optional specifying more parameters will result in a more accurate match?  Or at least guidance on a best practice here?

Reply:
Changes done as suggested